src/playbook.yaml

```diff
@@ -2,11 +2,90 @@
   become: yes
 
   tasks:
+
     - name: install system packages
       shell: "yes | aptdcon --hide-terminal --install 'build-essential'"
 
+    # Default python is 3.6, just install some libs
     - name: install python packages
       pip:
         name:
-          - synapseclient
+          - NumPy, pandas, synapseclient
         executable: pip3
+
+    - name: Install dependencies
+      shell: "yes | aptdcon --hide-terminal --install gdebi-core zlib1g-dev libclang-dev libssl-dev libffi-dev libcurl4-openssl-dev libapparmor1 libssl1.0.0 libxml2-dev libjpeg-dev libpng-dev libcairo2-dev libxt-dev bzip2 unixodbc unixodbc-devel"
+
+    - name: Add the CRAN apt key
+      apt_key:
+        keyserver: keyserver.ubuntu.com
+        id: E298A3A825C0D65DFD57CBB651716619E084DAB9
+
+    # R 3.6.3
+    - name: Install R 3.6.3
+      shell: "curl -O https://cdn.rstudio.com/r/ubuntu-1804/pkgs/r-3.6.3_1_amd64.deb && yes | gdebi r-3.6.3_1_amd64.deb"
+
+    - name: Add R-Project apt repository
+      shell: "aptdcon --add-repository 'deb https://cloud.r-project.org/bin/linux/ubuntu {{ ansible_distribution_release }}-cran35/' && aptdcon --refresh"
+
+    # Install R packages for R 3.6.3
+    - name: Install devtools
+      shell: "/opt/R/3.6.3/bin/R -e \"install.packages('devtools', repos=('https://cran.rstudio.com/'))\""
+
+    - name: Install synapser
+      shell: "/opt/R/3.6.3/bin/R -e \"install.packages('synapser', repos=c('http://ran.synapse.org', 'https://cran.rstudio.com/'))\""
+
+    - name: Install tidyverse
+      shell: "/opt/R/3.6.3/bin/R -e \"install.packages('tidyverse')\""
+
+    - name: Install shiny
+      shell: "/opt/R/3.6.3/bin/R -e \"install.packages('shiny', repos='https://cran.rstudio.com/')\""
+
+    - name: Install BiocManager
+      shell: "/opt/R/3.6.3/bin/R -e \"install.packages('biocManager', repos='https://cran.rstudio.com/')\""
+
+    - name: Install RStudio Professional Drivers
+      shell: "curl -O https://drivers.rstudio.org/7C152C12/installer/rstudio-drivers_1.6.0_amd64.deb && yes | gdebi rstudio-drivers_1.6.0_amd64.deb"
+
+    - name: Basic driver config
+      shell: "cp /etc/odbcinst.ini /etc/odbcinst.ini.bak && cat /opt/rstudio-drivers/odbcinst.ini.sample | tee /etc/odbcinst.ini"
+
+    # R 4.0.0
+    - name: Install R 4.0
```

Comment:
Why two versions of R?

Reply:
Backcompat (wondering if I should even add 3.4 to have the exact same version as the existing server). Then app owners can move on their schedule.